src/pages/MapPage/components/SearchBar/SearchBar.js

```diff
@@ -64,7 +64,16 @@ const SearchBar = React.forwardRef((props, ref) => {
     east: 121.0004439,
     west: 120.9953529
   }
+  // 24.787808334251203, 120.99299198044952
   const [positionName, setPositionName] = useState('')
+  const checkRange = (lat, lng) => {
+    if (lat < 24.790953568920005 && lat > 24.78259189570934) {
+      if (lng < 121.00204344319231 && lng > 120.99299198044952) {
```

Comment:
These values should be setup as constant values

Reply:
Okay already fix